src/ui/atoms/toast/Toast.tsx

```diff
@@ -30,48 +31,79 @@ type ToastProps = Readonly<{
    * Event handler called when the open state of the dialog changes.
    */
   readonly onOpenChange?: (isOpened: boolean) => void
+  /**
+   * Prevents the toast from closing automatically.
+   */
+  readonly preventAutoClose?: boolean
 }>
 
+// eslint-disable-next-line max-lines-per-function
 export const Toast: React.FC<ToastProps> = ({
   isOpened,
   autoDuration = 3000,
   severityLevel,
   title,
   description,
-  onOpenChange
+  onOpenChange,
+  preventAutoClose
 }: DeepReadonly<ToastProps>) => {
+  const closeButton = (
```

Comment:
You declare it as a value, and not a function as you did for renderTitle. It would be preferable to make the declaration consistent for all the fragments.